src/App.tsx

```diff
@@ -43,6 +44,7 @@ function App() {
               {/* alertID needs to be updated to display events */}
             </Route>
             <Route path="*" element={<Error404 />} />
+            <Route path="/eventdetails" element={<EventDetails />} />
```

Comment:
Make sure to pass in an event into the `EventDetails` component. I recommend just creating a dummy event and then passing it into this component.